packages/fx-core/src/component/utils/odrProvider.ts

```diff
@@ -25,6 +25,19 @@ export interface ODRTool {
 }
 
 export class ODRProvider {
+  /**
+   * Check if a server configuration is ODR-based.
+   * @param serverConfig The server configuration object
+   * @returns True if the server is ODR-based, false otherwise
+   */
+  static isODRServer(serverConfig: any): boolean {
+    if (serverConfig?.type !== "stdio" || !serverConfig.command) {
+      return false;
+    }
+    const configCommand = serverConfig.command;
```

Comment:
```suggestion
    const configCommand = serverConfig.command.toLowerCase();
```
So you only have to lower case once